tests/auth_tests/test_forms.py

```diff
@@ -261,6 +261,21 @@ class Meta(UserCreationForm.Meta):
             ['The password is too similar to the first name.'],
         )
 
+    def test_autocomplete(self):
+        form = UserCreationForm()
+
+        tests = (
+            ('username', 'username'),
+            ('password1', 'new-password'),
+            ('password2', 'new-password')
+        )
+        for field_name, autocomplete in tests:
+            with self.subTest(field_name=field_name, autocomplete=autocomplete):
+                self.assertEqual(
+                    form.fields[field_name].widget.attrs.get('autocomplete', ''),
```

Comment:
Use normal dictionary access instead of `.get()`. It is fine for us to blow up with a `KeyError` here and helps debugging because it is clearer whether the attribute is missing or the value is incorrect.